components/button/button.css

```diff
@@ -0,0 +1,179 @@
+    border-radius: 0.625rem;
+    border: none;
+    height: 2rem;
+    width: 8rem;
+    margin: 0.625rem;
+}
+
+.btn-primary{
+    background-color: var(--primary-color);
+    color: var(--light-color);
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color);
+    color: var(--light-color);
+}
+
+.btn-default{
+    background-color: var(--default-color);
+    color: var(--light-color);
+}
+
+.btn-success{
+    background-color: var(--success-color);
+    color: var(--light-color);
+}
+
+.btn-error{
+    background-color: var(--error-color);
+    color: var(--light-color);
+}
+
+.btn-disable{
+    background-color: var(--disabled-color);
+    color: var(--light-color);
+    cursor: not-allowed;
+}
+
+.btn-outline-primary{
+    border: 1px solid var(--primary-color);
+    color: var(--primary-color);
+}
+
+.btn-outline-secondary{
+    border: 1px solid var(--secondary-color);
+    color: var(--secondary-color);
+}
+
+.btn-outline-default{
+    border: 1px solid var(--default-color);
+    color: var(--default-color);
+}
+
+.btn-outline-success{
+    border: 1px solid var(--success-color);
+    color: var(--success-color);
+}
+
+.btn-outline-error{
+    border: 1px solid var(--error-color);
+    color: var(--error-color);
+}
+
+.btn-outline-disable{
+    border: 1px solid var(--disabled-color);
+    color: var(--disabled-color);
+    cursor: not-allowed;
+}
+
+.btn-link{
+    text-decoration: none;
+}
+
+.btn-link-primary{
+    border-bottom: 1px solid var(--primary-color);
+    color: var(--primary-color);
+    font-size: 1rem;
+    font-weight: 300;
+}
+
+.btn-link-secondary{
+    border-bottom: 1px solid var(--secondary-color);
+    color: var(--secondary-color);
+    font-size: 1rem;
+    font-weight: 300;
+}
+
+.btn-link-default{
+    border-bottom: 1px solid var(--default-color);
+    color: var(--default-color);
+    font-size: 1rem;
+    font-weight: 300;
+}
+
+.btn-float{
+    border-radius: 50%;
+    border: none;
+    height: 3rem;
+    width: 3rem;
+    z-index: 1;
+}
+
+.btn-icon > .fas, .btn-icon > .far{
+    padding-right: 0.625rem;
+}
+
+.btn:hover, .btn-float:hover, .btn-link:hover{
+    filter: brightness(90%);
+}
+
+.btn-outline-primary:hover{
+    background-color: var(--primary-color);
+    color: var(--light-color);
+}
+
+.btn-outline-secondary:hover{
+    background-color: var(--secondary-color);
+    color: var(--light-color);
+}
+
+.btn-outline-default:hover{
+    background-color:  var(--default-color);
+    color: var(--light-color);
+}
+
+.btn-outline-success:hover{
+    background-color:  var(--success-color);
+    color: var(--light-color);
+}
+
+.btn-outline-error:hover{
+    background-color: var(--error-color);
+    color: var(--light-color);
+}
+/* button class for the butto with icons*/
+.button{
+    display: inline-flex;
+    height:50px;
+    background-color: #009578;
+    border: none;
+    outline: none;
+    border-radius:5px ;
+    overflow: hidden;
+    font-size: 16px;
```

Comment:
Try to using one unit

Reply:
ok..i will take care of your feedback.. thank you for the feedback